.rubocop_todo.yml

```diff
@@ -74,12 +73,12 @@ Metrics/MethodLength:
     - 'app/lib/audit/checksum.rb'
     - 'app/lib/audit/moab_to_catalog.rb'
     - 'app/lib/preservation_catalog/s3_audit.rb'
-    - 'app/services/audit_results.rb'
     - 'app/services/checksum_validator.rb'
     - 'app/services/complete_moab_handler.rb'
     - 'app/services/moab_storage_root_reporter.rb'
     - 'app/services/moab_validator.rb'
-    - 'app/services/workflow_reporter.rb'
+    - 'app/services/reporters/event_service_reporter.rb'
+    - 'app/services/reporters/workflow_reporter.rb'
```

Comment:
I have a slight preference that this rubocop changes that are affecting master be done in a separate PR (or fixed if possible) as this PR isn't related to the two implicated files. But will defer to @sul-dlss/infrastructure-team thoughts on this.

Reply:
Thanks @aaron-collier, not sure the best way to handle when rubocop errors occur on code that the current PR didn't touch but needs to pass in order to be merged. I just ran `rubocop --auto-gen-config --auto-gen-only-exclude` to generate the changes in the *.rubocop_todo.yml* file.